packages/monaco/src/browser/monaco-command.ts

```diff
@@ -120,6 +127,93 @@ export class MonacoEditorCommandHandlers implements CommandContribution {
             }
         };
     }
+    protected newConfigIndentationHandler(): MonacoEditorCommandHandler {
+        return {
+            execute: (editor: MonacoEditor) => {
+                const configTabSize = (useSpaces: boolean) => {
```

Comment:
It should be a separate method, it should be called from `newConfigIndentationHandler` and `newConfigTabSizeHandler`.